BREEZE.rst

```diff
@@ -611,7 +617,13 @@ This is the current syntax for  `./breeze <./breeze>`_:
   -S, --static-check <STATIC_CHECK>
           Run selected static checks for currently changed files. You should specify static check that
           you would like to run or 'all' to run all checks. One of
-          [ all all-but-pylint bat-tests check-apache-license check-executables-have-shebangs check-hooks-apply check-merge-conflict check-xml debug-statements doctoc detect-private-key end-of-file-fixer flake8 forbid-tabs insert-license lint-dockerfile mixed-line-ending mypy pylint pylint-test setup-order shellcheck].
+
+  all all-but-pylint bat-tests check-apache-license
```

Comment:
The formatting/indentation here looks off -- it should be `"            "` I think (to put this "under" the option it's for, otherwise it's too high up)

Reply:
Resolved,